OJP/OJP_Trips.xsd

```diff
@@ -878,6 +895,172 @@
 			<xs:element name="TripFare" type="TripFareResultStructure" minOccurs="0" maxOccurs="unbounded"/>
 		</xs:sequence>
 	</xs:complexType>
+	<xs:complexType name="UserPreferencesStructure">
```

Comment:
I miss a safety related parameter here. OpenTripPlanner for example has bicycle safety.

Reply:
Stefan: safe vs fast. safety was more of not getting robbed below. 
Markus: Extrasafe is not really only about that. 
Stefan: What do you meand for safe? In OTP it is traffic safety. 
Markus: I meant accidents and crime.
Stefan: I suggest to split it into 
Safety:
* traffic safety. TrafficSafe
* feeling safe   xxx

Reply:
Alexandre: https://crimeandplace.com/